universal-login-react/src/core/utils/formatValue.ts

```diff
@@ -0,0 +1,5 @@
+import {utils} from 'ethers';
+import UniLoginSdk from '@unilogin/sdk';
+
+export const formatValue = (sdk: UniLoginSdk, amount: string, currency: string) =>
```

Comment:
it should get `(amount, currency, decimals)`

Reply:
then testing will be much more easier

Reply:
The intension was easy to use. Testing wasn't difficult